.github/workflows/visual-regression.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   visual-regression:
-    name: Run Tests
+    name: Run VRT Tests
```

Comment:
This should either say "Run VR Tests" or "Run VRT", since the T stands for Tests :)

Reply:
Yep, you are right. Updated.